guides.md

```diff
@@ -237,6 +239,140 @@ members. Skip the steps below at your discretion for low-stakes topics.
   1. Use the private dial-in number from our [technical Jitsi documentation](https://github.com/hyphacoop/organizing-private/blob/master/documents/infrastructure/jitsi.md).
   2. _After_ prompt, enter conference code `307 314 3734 #`
 
+## Onboarding Checklist
+
+This is a checklist for onboarding a new Member to the Co-operative 🚀
+
+### Setting up communications
+
+1. Get Signal number and add to group.
+
+1. Get Matrix account and invite to spaces:
+  - Private chat `#hyphacoop-private:tomesh.net`
+  - Public chat `#hyphacoop-open:tomesh.net`
+  - Community `+hyphacoop:tomesh.net`
+
+1. Set up [`hypha.coop` email](https://link.hypha.coop/email):
+  - Create new mailbox with Mailcow
```

Comment:
No block, but just cross-referencing to remember to update this once docs are PR'd in